kiali/kiali
# Fixed showing additional items for Service, fixed the REST logo

Issue https://github.com/kiali/kiali/issues/6665

Fixed the "REST API" logo.
Fixed Additional information showing for ServiceDetails page.

How to test it:
Make sure Kiali CR contains (by default contains, kiali configmap needs this):
```
additional_display_details:
  - annotation: kiali.io/api-spec
    icon_annotation: kiali.io/api-type
    title: API Documentation
```
Apply annotations into any of services:
```
kubectl annotate service kiali -n istio-system kiali.io/api-type=rest --overwrite
kubectl annotate service kiali -n istio-system kiali.io/api-spec=https://petstore.swagger.io/v2/swagger.json
```
Reload the Service list and details pages.
For testing in Workoads, the annotations need to be applied on workload template.

![Screenshot from 2023-11-30 15-06-38](https://github.com/kiali/kiali/assets/604313/ff64c69e-e445-4cd6-9a0a-042128bd6eb3)
![Screenshot from 2023-11-30 13-04-28](https://github.com/kiali/kiali/assets/604313/b3bb919a-2b5d-44ae-9cde-c9e7f2636f6d)
![Screenshot from 2023-11-30 13-04-11](https://github.com/kiali/kiali/assets/604313/56076c10-bbca-490a-9f98-490a0e28bb93)
![Screenshot from 2023-11-30 13-03-55](https://github.com/kiali/kiali/assets/604313/a1309d4a-b90b-4a0d-9326-da12953b0377)
